test/e2e/daemon_set.go

```diff
@@ -48,7 +48,14 @@ const (
 )
 
 var _ = Describe("Daemon set", func() {
-	f := &Framework{BaseName: "daemonsets"}
+	var f *Framework
+
+	AfterEach(func() {
```

Comment:
a part of me is wondering if Framework should allow passing a cleanup function or something which is guaranteed to run before Framework's afterEach cleanup function.